spec.html

```diff
@@ -19180,7 +19180,7 @@ <h1>Static Semantics: ContainsUseStrict</h1>
       <emu-see-also-para op="ContainsUseStrict"></emu-see-also-para>
       <emu-grammar>FunctionBody : FunctionStatementList</emu-grammar>
       <emu-alg>
-        1. If the Directive Prologue of |FunctionStatementList| contains a Use Strict Directive, return *true*; otherwise, return *false*.
+        1. If the Directive Prologue of |FunctionBody| contains a Use Strict Directive, return *true*; otherwise, return *false*.
```

Comment:
just to document my understanding: https://tc39.es/ecma262/#directive-prologue indicates that only FunctionBody, ScriptBody, or ModuleBody can have a Directive Prologue, so this is a bug fix.

Reply:
Correct.